semantic.cabal

```diff
@@ -99,7 +99,6 @@ library
                      , Analysis.PackageDef
                      -- Semantic assignment
                      , Assigning.Assignment
-                     , Assigning.Assignment.Deterministic
```

Comment:
Having JSON in a language package means we don’t need its assignment, and it was the only one using the `Deterministic` assignment API, so… 🔥